Source/Core/AudioCommon/AudioStretcher.h

```diff
@@ -1,27 +0,0 @@
-// Copyright 2017 Dolphin Emulator Project
-// SPDX-License-Identifier: GPL-2.0-or-later
-
-#pragma once
-
-#include <array>
-
-#include <SoundTouch.h>
```

Comment:
SoundTouch should be removed from Externals and the build system if we are no longer using it.